### PR TITLE
Tutorial - 4

### PR DESCRIPTION
**Pull request by ankitjavalkar**

### CODE REVIEW

File: tutorial_4_django_views_templates/slides.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+Tutorial: Create Views and Route your URLS(urls setting)
+===========================================
+[Demonstration time: 7 mins 00 s (0.817 ~ 82%) | Total time: 8 mins 34 s]
+
+Slide 1 [00:08 | 00:08]
+------------
+Title Slide
+**Creating Views and Routing URLs**
```
Comment (maheshgudi) [inline tutorial_4_django_views_templates/slides.md]: I think resolving fits better than routing.

File: tutorial_4_django_views_templates/slides.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+Slide 2 [00:12 | 00:20]
+--------------
+
+**Learning Objectives**
+
+In this tutorial, we will learn to;
```
Comment (maheshgudi) [inline tutorial_4_django_views_templates/slides.md]: colon, instead of semi colon

File: tutorial_4_django_views_templates/slides.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
+Tutorial: Create Views and Route your URLS(urls setting)
```
Comment (maheshgudi) [inline tutorial_4_django_views_templates/slides.md]: Maybe 'configuring urls' instead of 'urls setting'?

File: tutorial_1_intro/slide_1.md [hunk 4]
```diff
@@ -87,6 +87,7 @@ Slide 10 [00:12 | 01:27]
 ---------------
 
 **What is a virtual environment**
```
Comment (maheshgudi) [inline tutorial_1_intro/slide_1.md:89]: **What is a virtual environment?**

File: tutorial_4_django_views_templates/slides.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+  - Ubuntu 16.10
+  - Python 3.5 or higher version
+  - python3.4-venv
+  
+Slide 4 [00:11 | 00:42]
+---------------
+
+**Pre-requisites**
+
+In order to follow this tutorial, you need to know;
```
Comment (maheshgudi) [inline tutorial_4_django_views_templates/slides.md]: Maybe hyphen instead of semi colon?

File: tutorial_4_django_views_templates/slides.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+  - how to create models in django
+  - If not, see the relevant django tutorial on http://spoken-tutorial.org
```
Comment (maheshgudi) [inline tutorial_4_django_views_templates/slides.md]: AFAIK, models are covered in the previous video sessions; why not just point to those videos.

File: tutorial_4_django_views_templates/slides.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+
+Edit the /blog/views.py
+
+    # /blog/views.py
+    from django.http import HttpResponse
+    
+    from .models import Blog
+    
+    def get_blogs(request):
+        blogs = Blog.objects.all() # This is called a query
```
Comment (maheshgudi) [inline tutorial_4_django_views_templates/slides.md:54]: Technically, it is a queryset. But i understand the need to relate this with database queries. <code>blogs.query</code> would be the actually SQL query.

**Review by maheshgudi — COMMENTED** [review]
Please respond to the comments

File: tutorial_4_django_views_templates/slides.md (hunk 6 above)
Comment (ankitjavalkar) [reply tutorial_4_django_views_templates/slides.md]: Yes, links will redirect to the relevant tutorials. Currently links are acting as placeholders

File: tutorial_4_django_views_templates/slides.md (hunk 7 above)
Comment (ankitjavalkar) [reply tutorial_4_django_views_templates/slides.md:54]: Yes, the datatype is Queryset. But the statement in general is a query (language agnostic)

**Pull request merged**